api/v1/mapping/src/commonMain/kotlin/Mappings.kt

```diff
@@ -63,6 +63,7 @@ import org.eclipse.apoapsis.ortserver.api.v1.model.PluginConfiguration as ApiPlu
 import org.eclipse.apoapsis.ortserver.api.v1.model.ProcessedDeclaredLicense as ApiProcessedDeclaredLicense
```

Comment:
Commit message: The same applies as for the other commit message; capital letter at start of title and spurious whitespace.